### ISSUE TITLE
long sawscript file fails with too many open files

### ISSUE THREAD

**Issue by AndrewTolmach** [GaloisInc/saw-script#221]
Unpacking the attached zip and running `saw software.saw`  fails (at a slightly non-deterministic point) with

```
saw: z3: runInteractiveProcess: runInteractiveProcess: pipe: resource exhausted (Too many open files)
```

[long.zip](https://github.com/GaloisInc/saw-script/files/1216247/long.zip)

**Comment by brianhuffman** [comment]
The ecdsa proof in the saw-script examples directory has been giving me the same error lately as well.

**Comment by weaversa** [comment]
I think if you do something like

```
ulimit -c 0
```

everything works fine.

**Comment by weaversa** [comment]
Though, this is not the preferred solution.

**Comment by brianhuffman** [comment]
This appears to have been fixed by GaloisInc/cryptol@f02a3c7.
